src/reducers/related-views.ts

```diff
@@ -0,0 +1,10 @@
+import {Action} from '../actions/index';
+import {DEFAULT_RELATED_VIEWS, RelatedViews} from '../models/related-views';
+
+export function relatedViewsReducer(relatedViewToggler: RelatedViews = DEFAULT_RELATED_VIEWS,
+                                    action: Action): RelatedViews {
+  const {isHidden} = relatedViewToggler;
```

Comment:
Don't you have to check for action?  Wouldn't this mean that all actions will toggle the related view?